dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/Customer.cs

```diff
@@ -26,20 +26,10 @@
 
 namespace VisualLayerIntegration
 {
-    //Base customer object with a selection of data.
-    public class Customer
+    // Base customer object with a selection of data.
```

Comment:
>Base [](start = 7, length = 4)

Why "Base"? The comment is probably redundant because the class is so simple and obvious.